.github/ISSUE_TEMPLATE/platform-onboarding-checklist.md

```diff
@@ -0,0 +1,65 @@
+
+##  Checklist
+
+### EQIP / PIV / RA
+
+_EQIP - Electronic Questionnaires for Investigations Processing
+PIV - Personal Identity Verification
+RA - Remote Access_
+
+**If you're new to the VA, an EQIP submission is needed to obtain VA credentials and a PIV card**
+- [ ] Submit EQIP 
+- [ ] Obtain PIV card
+Note: AWS access can be granted once the EQIP has been submitted _before the PIV card is received_
+- [ ] Set up [TMS](https://www.tms.va.gov/secureauth35/SecureAuth.aspx) account and take required trainings
+- [ ] Ensure COR submits a request on your behalf for RA to the VA network 
+
+### Communications 
+
+**Platform coordination**
+- [ ] Share your calendar with VSP teams: va-vsp-team@oddball.io and  vsp-team.group@adhocteam.us
+- [ ] Ensure you're in Google Groups [va-vsp-team](https://groups.google.com/a/oddball.io/g/va-vsp-team) and [va-vsp-engineering](https://groups.google.com/a/oddball.io/g/va-vsp-engineering). 
+- [ ] Use `/support` in #vfs-platform-support to request access.
+- [ ] Ensure you have access to the Google Drive folder ["VSP team"](https://drive.google.com/drive/u/0/folders/0AJ6yVhfytxhFUk9PVA)
+- [ ] DSVA Slack Workspace
+  - [ ] #platform-cop-(frontend? devops?)
+  - [ ] #platform-team
+  - [ ] #vfs-all-teams
+
+**Team coordination**
+- [ ] [Meet the team](https://vfs.atlassian.net/wiki/spaces/TT2/pages/2401796102/Team+Charter+WIP#Meet-the-team)
+- [ ] [How we work](https://vfs.atlassian.net/wiki/spaces/TT2/pages/2401796102/Team+Charter+WIP#How-we-work-together)
+- [ ] [Current project](https://app.zenhub.com/workspaces/platform-tech-team-2-633efe4ca5a428e5294d7ade/issues/department-of-veterans-affairs/va.gov-team/47043)
+- [ ] [ZenHub Kanban Board](https://app.zenhub.com/workspaces/platform-tech-team-2-633efe4ca5a428e5294d7ade/board?repos=133843125,194202180) for Platform Tech Team 2
+
+### Documentation
+- [ ] [Request access to tools](https://depo-platform-documentation.scrollhelp.site/getting-started/Request-access-to-tools.969605215.html) documentation on the Platform Website
+- [ ] [Confluence](https://vfs.atlassian.net/wiki/home)
+  - [ ] [Platform Tech Team 2 space](https://vfs.atlassian.net/wiki/spaces/TT2/)
+  - [ ] [VA.Gov Platform Manual](https://vfs.atlassian.net/wiki/spaces/OT/pages/2354315287/Platform+Infrastructure+Manual+WIP)
+
+**Platform overview and design**
+- [ ] [Overview of the VA.Gov Platform](https://depo-platform-documentation.scrollhelp.site/getting-started/Welcome.1255276574.html)
+- [ ] [VA.Gov Platform system design and architecture](https://vfs.atlassian.net/wiki/spaces/OT/pages/2354315287/VA.Gov+Platform+Manual+WIP#System-design-and-architecture)
+
+
+### Developer tools 
+- [ ] GitHub
+  - [ ] Join the ['department-of-veterans-affairs' organization ](https://github.com/department-of-veterans-affairs)
+  - [ ] Get access to the [devops repo](https://github.com/department-of-veterans-affairs/devops)

```

Comment:
Maybe something like: "Get access to the following repos: [vets-website](https://github.com/department-of-veterans-affairs/vets-website), [content-build](https://github.com/department-of-veterans-affairs/content-build/), [devops](https://github.com/department-of-veterans-affairs/devops), [vets-api](https://github.com/department-of-veterans-affairs/vets-api)"


While I don't think all of them will be used, these are the core used by many

Reply:
![image](https://user-images.githubusercontent.com/125288357/224147846-d2487420-f270-47cb-bb8d-ffdc6ef93d53.png)